libs/src/index.ts

```diff
@@ -1,7 +1 @@
-export { sdk } from './sdk'
-
-/**
- * @deprecated Please migrate to using `sdk` when possible
- */
-export * from './AudiusLibs'
```

Comment:
Doesn't this break consumers that aren't using the legacy import? Like this for example https://github.com/AudiusProject/audius-protocol/blob/master/creator-node/src/routes/files.js#L40

Reply:
Oh yeah you're so right, adding the ship stuff back